frontend/src/styles/tailwind.css

```
@@ -1,4 +1,6 @@
 @import 'tailwindcss' source(none);
 @source '../**/*.{js,ts,jsx,tsx}';
 
-@import 'tw-animate-css';
\ No newline at end of file
+@import 'tw-animate-css';
+
+@custom-variant mob-land (@media (orientation: landscape) and (max-height: 500px));
```

Stylelint (17.10.0)

[error] Unexpected unknown at-rule "@custom-variant" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)